src/plugins/intel_gpu/tests/functional/single_layer_tests/reduce.cpp

```diff
@@ -0,0 +1,101 @@
+
+using ov::test::InputShape;
+using ReduceInputParams = std::tuple<
+                            ov::Shape,                // Input shapes
+                            ov::element::Type,        // Input precision
+                            std::vector<int64_t>,     // Reduce Axes
+                            bool>;                    // Keep dims
+
+
+class ReduceSumSqueezeTest : public testing::WithParamInterface<ReduceInputParams>,
+          virtual public ov::test::SubgraphBaseStaticTest {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<ReduceInputParams> obj) {
+        ov::Shape input_shape;
+        ov::element::Type input_precisions;
+        std::vector<int64_t> reduce_axes;
+        bool keep_dims;
+
+        std::tie(input_shape, input_precisions, reduce_axes, keep_dims) = obj.param;
+
+        std::ostringstream result;
+        result << "IS=ReduceSum_";
+        result << ov::test::utils::vec2str(input_shape) << "_";
+        result << "reduce_axes=" << ov::test::utils::vec2str(reduce_axes) << "_";
+        result << "keep_dims=" << keep_dims << "_";
+        result << "input_precision=" << input_precisions;
+        return result.str();
+    }
+
+protected:
+    ov::Shape input_shape;
+    std::vector<int64_t> reduce_axes;
+    bool keep_dims;
+
+    void SetUp() override {
+        targetDevice = ov::test::utils::DEVICE_GPU;
+        core->set_property(targetDevice, {ov::enable_profiling(true)});
+
+        ov::element::Type input_precision;
+        std::tie(input_shape, input_precision, reduce_axes, keep_dims) = GetParam();
+
+        ov::Shape add_val_shape = {input_shape.begin(), input_shape.end() - 1};
+        ov::test::utils::InputGenerateData in_data;
+        in_data.start_from = -10;
+        in_data.range = 10;
+        auto add_val_tensor = ov::test::utils::create_and_fill_tensor(input_precision, add_val_shape, in_data);
+
+        auto input_node = std::make_shared<ov::op::v0::Parameter>(input_precision, input_shape);
+        auto axes_node = ov::op::v0::Constant::create(ov::element::i64, {1}, reduce_axes);
+        auto reduce_node = std::make_shared<ov::op::v1::ReduceSum>(input_node, axes_node, keep_dims);
+        reduce_node->set_friendly_name("ReduceSum");
+        auto add_val_node = std::make_shared<ov::op::v0::Constant>(add_val_tensor);
+        auto add_node = std::make_shared<ov::op::v1::Add>(reduce_node, add_val_node);
+        auto result = std::make_shared<ov::op::v0::Result>(add_node);
+        function = std::make_shared<ov::Model>(result, ov::ParameterVector{input_node}, "input");
+    }
+
+    void run() override {
+        ov::test::SubgraphBaseStaticTest::run();
+        auto profiling_info = inferRequest.get_profiling_info();
+        bool found = false;
+        for (auto& info : profiling_info) {
+            if (info.node_name == "ReduceSum_reorder" &&
+                info.status == ov::ProfilingInfo::Status::OPTIMIZED_OUT)
+                found = true;
```

Comment:
I think this check is too tightly coupled with current implementation. Maybe is it possible to check output dimension of reduce layer (or `Add` layer)?

Reply:
It has changed to compare the shape of output.